### PR TITLE
New command `slcli bandwidth pools-edit`

### PR DESCRIPTION
**Pull request by edsonarios**
Issue #1989 
```
softlayer-python/ (issue1989) $ slcli bandwidth pools-edit 1234567 --name TestRegionEdited
┌──────────────┬──────────────────┐
│     Name     │      Value       │
├──────────────┼──────────────────┤
│      Id      │     1234567      │
│  Name Pool   │   TestRegionEdited │
│    Region    │ SNG/HKG/OSA/TOK  │
│ Created Date │ 2023-05-31 07:13 │
└──────────────┴──────────────────┘
```

### CODE REVIEW

File: tests/CLI/modules/bandwidth_tests.py [hunk 1]
```diff
@@ -56,6 +56,16 @@ def test_create_bandwidth(self):
         self.assertEqual('NewRegion', json_output['Name Pool'])
         self.assertEqual('SJC/DAL/WDC/TOR/MON', json_output['Region'])
 
+    def test_edit_bandwidth(self):
+        result = self.run_command(['bandwidth', 'pools-edit', '123456', '--name=MexRegionEdited'])
+        self.assert_no_fail(result)
+        self.assert_called_with('SoftLayer_Network_Bandwidth_Version1_Allotment', 'editObject')
+        json_output = json.loads(result.output)
+        pp(json_output)
```
Comment (BrianSantivanez) [inline tests/CLI/modules/bandwidth_tests.py:64]: Remove this line

Comment (ramkishor-ch) [inline tests/CLI/modules/bandwidth_tests.py:64]: Hi @edsonarios 

Can you please **provide** the **answers** to the following **questions**.

**Questions:**
1. What is pp() , line number: 64?
2. Why pp() is required, line number: 64?

Thanks,
Ramkishor Chaladi.

Comment (edsonarios) [reply tests/CLI/modules/bandwidth_tests.py]: pp it's to format better the output
only is required in develop time, now was removed

**Comment by JayasilanJibm** [comment]
Hi,

I have tested the code and reviewed the changes. Approving the PR.
Please find the screenshot for further reference.

Screenshot 2023-06-19 at 8 14 03 PM

Screenshot 2023-06-19 at 8 14 24 PM

Thanks,
Jayasilan

**Review by allmightyspiff — APPROVED** [review]

**Comment by allmightyspiff** [comment]
@ramkishor-ch 

> slcli bandwidth pools-edit 1234567 --name TestRegionEdited
> slcli bandwidth pools-edit **--id** 1234567 --name TestRegionEdited

As a general pattern, the ID of any object you interact with on the SLCLI should be a [Click Argument](https://click.palletsprojects.com/en/8.1.x/arguments/#arguments) which means you specify it after the command without any dashes. Having a `--id` flag might let the users think that it was optional, or that they might be able to change the id (which you can't do). 

So, when writing a command you should always have the REQUIRED options as arguments, and any options as parameters

**Pull request merged**